shadowsheep1/io-app
# [1] Creation of a new profile page with header with back button, contextual help and title.

## Short description
This PR creates a new user profile page as specified in #1.
- The user can navigate to this page using the same "Profile" tab section in the current main `BottomBar`.

## List of changes proposed in this pull request

- This page has an header with the navigation back button and an icon to open the contextual help showing a generic profile help information.
- This page shows only the user profile's **Name and Surname**, **Fiscal Code** and **Email Address** of the user profile. It's simpler than the previous one. The above user profile's data are shown only if they are present.

| Old Profile Page  | New Profile Page |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/16268789/201059565-71e7b36b-ca12-4d9c-aa80-c3876f679af7.png" width="150" />  |   <img src="https://user-images.githubusercontent.com/16268789/201059556-c81d4b73-d451-497a-9637-0a9d405b6f87.png" width="150" /> |

By tapping on the **contextual help** icon in this page you should see this FAQ Section:

| FAQ screen shown by tapping <br/> on the contextual help icon|
| ------------- | 
|<img src="https://user-images.githubusercontent.com/16268789/201069274-a9c7cad1-b3d7-4abb-b0f7-c852b16de75c.png" width="150" />|

## How to test
- Run the app and then navigate to the new profile page by tapping the profile icon on the bottom bar. What you see is the new profile pages shown above. 
- By tapping on the **back button** you should go back to the previous screen.
- By tapping on the **contextual help icon** you should see the above FAQ screen.

There is a **Jest** file test to run called: `ProfileMainScreen2.test.tsx`.
